infra/gcp/tests/gke.tf

```diff
@@ -0,0 +1,51 @@
+// WARNING, MAKE SURE YOU DON"T DESTROY THESE CLUSTERS ACCIDENTALLY
+
+
+module "prow_trusted" {
+  source                     = "terraform-google-modules/kubernetes-engine/google"
+  project_id                 = module.project.project_id
+  name                       = "prow-trusted"
+  regional                   = false
+  zones                      = ["us-central1-a"]
+  release_channel            = "STABLE"
+  network                    = "default"
+  subnetwork                 = "default"
+  ip_range_pods              = "gke-prow-trusted-pods-f579223d"
+  ip_range_services          = "gke-prow-trusted-services-f579223d"
+  http_load_balancing        = true
+  network_policy             = false
+  horizontal_pod_autoscaling = true
+  filestore_csi_driver       = false
+  create_service_account     = false
+
+  node_pools = [
```

Comment:
I saw you've already switched to the new node pool, which is nice! 

I have a general question about terraform, when we run `terraform apply` to reconcile the config changes like this, will there be downtime? Or does terraform always follow steps like https://cloud.google.com/kubernetes-engine/docs/tutorials/migrating-node-pool to ensure there is minimal impact?

Reply:
> I have a general question about terraform, when we run `terraform apply` to reconcile the config changes like this, will there be downtime? Or does terraform always follow steps like https://cloud.google.com/kubernetes-engine/docs/tutorials/migrating-node-pool to ensure there is minimal impact?

No. You need to create a new nodepool and then cordon & drain manually. For this particular cluster, it is not busy so making the new one and deleting the old one right away is fine. For Prow/Prow Build we will need to cordon + drain for much longer

Reply:
Got it, thanks for the explanation!